internal/app/vodtester/vodtester_app.go

```diff
@@ -144,11 +145,107 @@ func (vt *vodTester) Start(fileUrl string, taskPollDuration time.Duration) (int,
 		}
 	}
 
+	filePath := "/tmp/uploadtester.mp4"
+	resp, err := http.Get(fileUrl)
+
+	if err != nil {
+		glog.Errorf("Error downloading fileUrl=%s err=%v", fileUrl, err)
+	}
+	defer resp.Body.Close()
+
+	out, err := os.Create(filePath)
+	if err != nil {
+		glog.Errorf("Error creating file filePath=%s err=%v", filePath, err)
+	}
```

Comment:
Missing returns after handling the errors (tho this code should go away if you replace with the built-in file).